src/components/DownloadForm/Controls/index.js

```diff
@@ -109,7 +114,24 @@ export class Controls extends PureComponent /*:: <Props> */ {
             Please generate the file in order to download it.
           </div>
         ) : null}
-
+        {interProVersion > version && (
+          <div className={f('callout', 'alert', 'withicon')}>
+            <h5>The existing download is outdated.</h5>
+            <p>
+              It was obtained using InterPro version <i>{version}</i>, while the
+              current version is <i>{interProVersion}</i>. You need to remove
+              the existing file before re-generating the download.
+            </p>
+            <p>
+              <Link
+                className={f('button', 'hollow')}
+                onClick={this._handleCancelClick}
+              >
+                Remove the current download
+              </Link>
+            </p>
+          </div>
```

Comment:
Like good wine, it would only happen with time... 😆

You are right to say that `version` comes from the `InterPro-Version` header, but then it's saved in IndexedDB so eventually they can be out of sync

Reply:
I didn't know the `InterPro-Version` header was also cached. Now it makes sense.